### PR TITLE
ziafazal/YONK-345: renamed organizations app

### PR DESCRIPTION
**Pull request by ziafazal**
This PR has changes necessary to rename `organizations` app to `edx_solutions_organization`. It has a django management command `rename_organizations_app` to make database changes like renaming of tables.
@msaqib52 would you please review?

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -17,5 +17,5 @@ Open edX Platform Integration
 
 .. code-block:: bash
 
-   $ python manage.py lms --settings test test organizations.tests
+   $ paver test_system -s lms -t organizations
```
Comment (msaqib52) [inline README.rst]: i think app name should also be updated here to new name: `paver test_system -s lms -t edx_solutions_organizations`

File: edx_solutions_organizations/management/commands/rename_organizations_app.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+def get_table_names():
+    """
+    Returns list of table name organization app has
+    """
+    return [
+        'organization_workgroups',
+        'organization_groups',
+        'organization_users',
+        'organization',
```
Comment (msaqib52) [inline edx_solutions_organizations/management/commands/rename_organizations_app.py:20]: why we are not renaming `organizationgroupuser` table?

**Comment by ziafazal** [comment]
@msaqib52 Thanks for the feedback made changes as suggested.

**Comment by msaqib52** [comment]
👍

**Pull request merged**